packages/apps/plugins/plugin-client/src/components/ClientSettings.tsx

```diff
@@ -18,6 +20,27 @@ export const ClientSettings = ({ settings }: { settings: ClientSettingsProps })
       <SettingsValue label={t('enable experimental automerge backend')}>
         <Input.Switch checked={settings.automerge} onCheckedChange={(checked) => (settings.automerge = !!checked)} />
       </SettingsValue>
+      <SettingsValue label={t('choose storage adaptor')}>
+        <Select.Root
+          value={settings.storageAdapter}
+          onValueChange={(value) => {
+            settings.storageAdapter = value === 'opfs' ? 'opfs' : 'idb';
+          }}
+        >
+          <Select.TriggerButton />
+          <Select.Portal>
+            <Select.Content>
+              <Select.Viewport>
+                {StorageAdapters.map((adapter) => (
+                  <Select.Option key={adapter} value={adapter}>
+                    {t(`settings storage adaptor ${adapter} label`)}
```

Comment:
Is it possible to put a warning here: "Warning: swapping the storage adapter will make your data unvailable."